kafka/librdkafka_vendor/import.sh

```diff
@@ -48,7 +48,7 @@ if [[ -z $version ]]; then
 fi
 
 # Verify branch state
-curr_branch=$(git symbolic-ref HEAD 2>/dev/null | cut -d"/" -f 3)
+curr_branch=$(git symbolic-ref HEAD 2>/dev/null | cut -d"/" -f 3-)
```

Comment:
Why this change? Can there be slashes in branches?

Reply:
Yup, there definitely can - in fact this branch name has a slash in it too (`ktsanaktsidis/fix_dynamic_build`). At Zendesk it's conventional to structure branch names as `<your name>/<issue number>/<branch description>`, so I did this out of habit and found that the import script couldn't handle it.